packages/typespec-test/test/NetworkAnalytics.Management/generated/typespec-ts/src/models/models.ts

```diff
@@ -389,8 +389,8 @@ export enum KnownManagedServiceIdentityType {
   SystemAssigned = "SystemAssigned",
   /** UserAssigned */
   UserAssigned = "UserAssigned",
-  /** SystemAssigned, UserAssigned */
-  "SystemAssigned, UserAssigned" = "SystemAssigned, UserAssigned",
+  /** SystemAndUserAssigned */
+  SystemAndUserAssigned = "SystemAssigned, UserAssigned",
```

Comment:
I remember we generate the previous way in HLC, @kazrael2119 could you help double check?

Reply:
see issue https://github.com/Azure/autorest.typescript/issues/2772